src/jit/gentree.h

```diff
@@ -1468,6 +1463,9 @@ struct GenTree
     // yields an address into a local
     GenTreeLclVarCommon* IsLocalAddrExpr();
 
+    // Like IsLocalAddrExpr but local or field, not address-of
+    bool IsLocalExpr(Compiler* comp, GenTreeLclVarCommon** pLclVarTree, FieldSeqNode** pFldSeq);
+
```

Comment:
This function isn't that similar to  IsLocalAddrExpr,
Instead the comment should read
// Determine if this is a LclVarCommon node and return some additional info about it in the two out parameters